FAIL
run: wait(exec("multi-recurse")) returned -1: FAILED
